frontend/openchat-shared/src/domain/chat/chat.ts

```diff
@@ -709,7 +709,6 @@ export type LocalChatSummaryUpdates = {
     updated?:
         | {
               kind?: undefined;
-              latestMessage?: EventWrapper<Message>;
```

Comment:
We merge unconfirmed messages into the chat summary so this was never actually needed. Also, we were only setting it in one place which has now been updated to instead use `updateSummaryWithConfirmedMessage`, because it is called within `addServerEventsToStores` at which point the message is confirmed.